lnwire/lnwire.go

```diff
@@ -171,12 +171,35 @@ func writeElement(w io.Writer, element interface{}) error {
 			}
 		}
 	case *btcec.Signature:
+		// Serialize the signature with all the checks that entails
 		sig := e.Serialize()
-		if len(sig) > 73 {
-			return fmt.Errorf("Signature too long!")
+		var b [64]byte
+		// Extract lengths of R and S
```

Comment:
In general having the format of the serialized sig: 
````
0x30 <length> 0x02 <length r> r 0x02 <length s> s
```` 

displayed somewhere in the comment can be helpful as it's easier to follow the logic with the full format in mind.